docs/authenticating-torus-users/torus-users.mdx

```diff
@@ -0,0 +1,95 @@
+---
+title: Torus Users
+hide_table_of_contents: true
+id: torus-users
+---
+
+#### Getting the Challenge from the AuthJS Backend
+
+For a user that signed up with `Google` with the email `hello@tor.us`, with the share `0xBEEF4321...` wanting to receive a challenge on app scoped key
+`0x1234C0DE...`
+
+##### POST https://auth-js-backend.tor.us/challenge/get
+
+```json
+{
+  "verifier": "google",
+  "verifier_id": "hello@tor.us",
+  "oauth_pub_key": "0xBEEF4321...",
+  "app_scoped_pubkey": "0x1234C0DE...",
+  "network": "mainnet"
+}
+```
+
+##### Challenge Generation
+
+1. Store the following data in a database / cache
```

Comment:
why we need to write internal working of challenge generation part here? This should be internal right? We should simply write endpoint of getting challenge with req and res params explanation.

Reply:
Makes sense.. It can be abstracted from the user..